httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java

```diff
@@ -130,6 +188,17 @@ private HttpHeaders() {
 
     public static final String STATUS_URI = "Status-URI";
 
+    /**
+     * The HTTP {@code Set-Cookie} header field name.
+     * @see <a href="https://tools.ietf.org/html/rfc2109#section-4.2.2">Section 4.2.2 of RFC 2109</a>
```

Comment:
@arturobernalg I am not sure about references to RFCs in the source code. For instance, HttpClient conforms to rfc6265 and not rfc2109. I would not want to scan the source code for RFC references every time protocol level gets upgraded.

Reply:
Agree, changed